client/my-sites/plans/jetpack-plans/product-lightbox/quantity-dropdown.tsx

```diff
@@ -62,7 +69,16 @@ const QuantityDropdown: FC< QuantityDropdownProps > = ( { product, siteId, onCha
 	return (
 		<div className="product-lightbox__variants-dropdown">
 			<FormFieldset>
-				<FormLegend>{ PRODUCT_OPTIONS_HEADER[ product?.productSlug ] }</FormLegend>
+				<FormLegend>
+					<span>{ PRODUCT_OPTIONS_HEADER[ product?.productSlug ] }</span>
+					<InfoPopover
+						position="right"
+						screenReaderText={ translate( 'Learn more' ) }
+						className="product-lightbox__variants-dropdown--popover"
+					>
+						{ PRODUCT_OPTIONS_HEADER_INFO_CONTENT[ product?.productSlug ] }
```

Comment:
We might want to use `preventWidows()` formatting function - currently I see the widow on the screenshot attached to the PR.

Reply:
Nice suggestion for improvement thank you!